pkg/hvs/controllers/flavortemplate_controller.go

```diff
@@ -119,16 +126,19 @@ func (ftc *FlavorTemplateController) Search(w http.ResponseWriter, r *http.Reque
 	defaultLog.Trace("controllers/flavortemplate_controller:Search() Entering")
 	defer defaultLog.Trace("controllers/flavortemplate_controller:Search() Leaving")
 
-	paramIncludeDelete := r.URL.Query().Get("include_deleted")
+	if err := utils.ValidateQueryParams(r.URL.Query(), flavorTemplateSearchParams); err != nil {
```

Comment:
Didn't get where "ValidateQueryParams" function is defined  in this MR

Reply:
"ValidateQueryParams" method is defined in  intel-secl/pkg/hvs/utils/controller.go